bescort.lic

```diff
@@ -222,6 +222,8 @@ class Bescort
       take_rh_lang_barge
     elsif args.shard_gates
       use_shard_gates
+    elsif args.thief_guild
+	  enter_thief_guild
```

Comment:
Please follow the indentation of the other options (and it looks like there is a tab here, not spaces?)